migrations/2025-07-29-144525_add_docs_ipfs_hash_to_uploads/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add docs_ipfs_hash column to uploads table
+ALTER TABLE uploads ADD COLUMN docs_ipfs_hash VARCHAR DEFAULT NULL;
```

Comment:
nit: maybe worthwhile declaring max length for this field (i.e. length of an ipfs hash?)
```suggestion
ALTER TABLE uploads ADD COLUMN docs_ipfs_hash VARCHAR(100);
```